packages/components/src/playground/button.stories.tsx

```diff
@@ -1,30 +1,35 @@
 import * as React from "react";
-
 import Button from "./button";
-import { action } from "@storybook/addon-actions";
+import { Story } from "@storybook/react/types-6-0";
```

Comment:
Normally could write

```ts
import type { Story } from ...
```

But our eslint parsing doesn't seem to be able handle that 🤔

Reply:
thanks for flagging